backend/controller/leases/dal/lease.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/google/uuid"
 
 	"github.com/TBD54566975/ftl/backend/controller/leases"
-	"github.com/TBD54566975/ftl/backend/controller/sql"
+	parentsql "github.com/TBD54566975/ftl/backend/controller/sql"
```

Comment:
Shouldn't this be using `leases/sql`?

Reply:
This was also related to promoting all the refactored `Queries` structs in `backend/controller/sql/conn.go` for the `Tx` behavior. Basically, instead of giving each `sql` package its own `conn.go`, they all get lumped under the same parent package's `DBI` struct.

Reply:
I see. We don't want to do this though, we want them to be completely decoupled.

Reply:
removed with the latest change :)